src/html/chartAuthoringInterfaceTemplate.html

```diff
@@ -1,2 +1,4 @@
+<div class="floec-chartTitle"><p class="flc-inlineEdit-text"></p></div>
```

Comment:
As specified in earlier comments, if both styling and js controlling are needed for a container, two class names should be used, and the controlling css comes before styling css(es). For example,

```
<div class="floec-chartTitle floe-chartTitle"><p class="floec-inlineEdit-text floe-inlineEdit-text"></p></div>
``` 

BTW, shall we take the chance to use ```floec-inlineEdit-text``` instead of ```flc-inlineEdit-text``` since it's redefined anyway?